sos/report/plugins/nvidia.py

```diff
@@ -44,4 +53,5 @@ def setup(self):
         )
         self.add_journal(boot=0, identifier='nvidia-persistenced')
 
+
```

Comment:
nit, do we need the extra line here? not overly precious about it though

Reply:
Good point, I think we can remove it. 
My only doubt then is what to do with this add_journal() call before it. The service is now in the services tuple, but the boot=0 is what makes me doubt if it's safe or not to remove it. Any thoughts on this?